<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagePlayers</class>
 <widget class="QMainWindow" name="ManagePlayers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>616</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="return_button">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="new_player_button">
      <property name="text">
       <string>New Player</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Manage Players</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableWidget" name="players_table_widget">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>112</number>
      </attribute>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>League Rank</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Avg Season Score</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Avg Lifetime Score</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number of 180s</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string># Season Turns</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Remove</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
